### ISSUE TITLE
Accessibility: Unable to view what a message is in reply to.

### ISSUE THREAD

**Issue by TheQuinbox** [TelegramMessenger/Telegram-iOS#683]
### Checklist
- [x] I am reporting an issue in existing functionality that does not work as intended
- [x] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
Somewhat related to #504. Accept now, you get no feedback at all what something's in reply to. 
### Expected Behavior
Blind people should have a way to tell what a message is in reply to, either with a 3 finger tap like #504 suggested, or what I'd personally prefer, a VoiceOver jesture.
### Actual Behavior
We have no way of telling what a message is in reply to.
### Steps to Reproduce
1. Start VoiceOver
2. Go to a message that's a reply
3. Notice that the reference is not read.

### Environment
**Device:** iPhone 13 Pro Max (256 GB)

**iOS version**: 15.1.1

**App version:** Latest in App Store

**Comment by TheQuinbox** [comment]
Little correction here, my prefered way would be a VoiceOver action on the Roter. So I could swipe down, tap on view reference, and it would read.
